ui/package.json

```diff
@@ -30,6 +30,7 @@
     "node-sass": "6.0.1",
     "numeral": "^2.0.6",
     "papaparse": "^5.1.0",
+    "prettier": "^2.7.1",
```

Comment:
We don’t really distinguish between development dependencies and normal dependencies, so I simply added Prettier here.